src/components/GridTile.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 import moment from 'moment';
+import { Link } from 'react-router-dom';
 
 const GridTile = props => {
   return (
-    <a className="gridtile-container" href={`article/${props.slug}`}>
+    <a className="gridtile-container">
```

Comment:
Let's just apply the `className` to the `Link` component and get rid of the `<a>` tag.